tests/RobotFramework/tests/cumulocity/configuration/configuration_with_file_transfer_https.robot

```diff
@@ -79,18 +87,27 @@ Get Configuration Should Succeed
 Get Configuration Should Fail
     [Arguments]    ${failure_reason}    ${device}    ${external_id}
     Cumulocity.Set Device    ${external_id}
-    ${operation}=    Cumulocity.Get Configuration    CONFIG1
+    ${operation}=    Cumulocity.Get Configuration    tedge-configuration-plugin
     Operation Should Be FAILED    ${operation}    failure_reason=${failure_reason}   timeout=120
 
 Update Configuration Should Fail
     [Arguments]    ${failure_reason}    ${device}    ${external_id}
     Cumulocity.Set Device    ${external_id}
     Cumulocity.Should Support Configurations    tedge-configuration-plugin    /etc/tedge/tedge.toml    system.toml    CONFIG1    CONFIG1_BINARY
     ${config_url}=    Cumulocity.Create Inventory Binary    tedge-configuration-plugin    tedge-configuration-plugin    file=${CURDIR}/tedge-configuration-plugin-updated.toml
-    ${operation}=    Cumulocity.Set Configuration    CONFIG1    ${config_url}
+    ${operation}=    Cumulocity.Set Configuration    tedge-configuration-plugin    ${config_url}
     Operation Should Be FAILED    ${operation}    failure_reason=${failure_reason}   timeout=120
     Cumulocity.Should Support Configurations    tedge-configuration-plugin    /etc/tedge/tedge.toml    system.toml    CONFIG1    CONFIG1_BINARY
 
+Update Configuration Should Succeed
+    [Arguments]    ${device}    ${external_id}
```

Comment:
Technically speaking we don't need the `${device}` argument since it is not used here (unless I'm missing something), but since other tests also have unused arguments, then I won't block because of it.